.github/workflows/build-opencv-base-container.yml

```diff
@@ -10,24 +10,68 @@ on:
     - build/intermediate-containers.mk
     - Makefile
   pull_request:
+    types: [assigned, opened, synchronize, reopened, labeled, unlabeled]
     branches: [ main ]
     paths:
     - .github/actions/build-intermediate/**
     - .github/workflows/build-opencv-base-container.yml
     - build/containers/intermediate/Dockerfile.opencvsharp-build
     - build/intermediate-containers.mk
     - Makefile
+  pull_request_review_comment:
+    types: [created]
 
 env:
   AKRI_COMPONENT: opencvsharp-build
   MAKEFILE_COMPONENT: opencv-base
 
 jobs:
 
+  add-label-missing-comment:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'build-dependency-containers') == false
+    steps:
+    - uses: actions/checkout@v2
+    - name: Add comment about missing build-dependency-containers label
+      uses: actions/github-script@0.9.0
+      if: success()
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
+        script: |
+          github.issues.createComment({
+            issue_number: context.payload.pull_request.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: 'hey this apparently should run but running it will take hours. Do you want to run it? If so, add flag build deps cont. (also tags Akri Maintainers)'

```

Comment:
I think this comment may need to be updated. Does this tag the maintainers? I remember you mentioning that that is not longer necessary as the workflow will leave a comment for anyone to add the label. Also, can this message be formalized a little?

Reply:
I updated the comment a little bit. This comment won't tag the maintainers. Anyone can add labels using /add comment. 
Is this formalized enough? Any suggestions? Thanks!